### DISCUSSION TITLE
[Design] Nuclear gamemode rework

### DISCUSSION THREAD

**Discussion by datlo** [ParadiseSS13/Paradise#21098] [Documents]
Nuke ops is a pretty good antag, but the game mode itself could use freshening up. The goals of these changes are : 

1. Making it clearer who won and who lost, with no arcane conditions that almost require code diving to understand the mode.
2. Give the crew a chance to actually disarm the nuke by forcing the nukies to plant it close to station center for a successful detonation.
3. Make war rounds follow a 'Last stand' story structure where the crew must survive the assault until CC reinforcements arrive.

Changes to the game mode :

Victory conditions : 

1. Syndicate Victory : Station succesfully destroyed. Loss of personnel irrelevant, even if the squad is wiped, a succesful detonation is a victory.
2. Nanotrasen Victory : Any other situation. Even if the NAD is not secure by the time the round ends, this is NT territory and they can send more reinforcements : Nukies should not take their sweet time nuking the station.

The syndicate nuke : 

1. Is now a weaker version of the standard NT nuke that requires deployment on station structural weakness. These are 3 rooms randomly selected that are close to the station center.
2. If the nuke detonates outside of these structural weakness rooms, it will cause a massive explosion but not destroy the station. The round will not end immediately but the shuttle will be accelerated to finish the round faster.
3. Arming the nuke now triggers a CC announcement, informing the crew they need to move to disarm the nuke immediately. 
4. Arming the nuke now scrambles the escape shuttle, preventing it from leaving, but not arriving. The crew must disarm or move the nuke before escaping. 

War : 
 1. Declaring war will now have CC send the escape shuttle with a 40 minute timer, and inform the crew that reinforcements are en route and to survive until then.
 2. During war, the shuttle comes packed with an ERT, partially scaled with remaining nuke ops strength, to assist with evacuation and safeguard the NAD.
 3. If the nuke ops do not successfully detonate the nuke before the shuttle leaves the station, the round ends there : NT shock troops board the station and wipe out any remaining nuke ops onboard (Not pictured ingame as the round is over.)
 4. Due to the changes to the game mode that favors crew, nuke ops might get a small amount more TC from war declaration as compensation. Will need testing.
 
The standard NT nuke is not touched by these changes.

These changes altogether should hopefully lead to more interesting rounds, with the crew able to still fight back if the NAD holder gets taken out quickly.

**Comment by Edan52** [comment]
My main concern with this is the conflict between having the nuke have to be detonated close to the centre of the station, and the nuke ops also escaping. This will encourage them to be more of a suicide squad as they hold their ground until the nuke is detonated since the win condition favours that, and since the nuke will be much easier for the crew to find now. Maybe that is your intent? There is also the random nature of the rooms that are chosen, with some potentially being easier than others.

My other concerns that I don't particularly mind much are that a syndicate nuke triggering delta doesn't make much sense but I see why you have done it from a gameplay point of view. I also don't think the extra TC will change much because they already get a ridiculous amount.

**Reply by datlo** [reply]
I also like the nuke ops escape into the sunset as the station explodes behind them so it's regrettable that at least some ops will have to stay behind, but it's a necessity to have the nuke be interactable with. I'm open to suggestions if you have any better ideas on that point.

The delta alert can be replaced by a CC announcement that an unauthorized live nuclear bomb is onboard the station instead, this might better fit yeah.

**Comment by Sadhorizon** [comment]
Hm, one question - how would that influence stealth ops? The announcement about nuking sounds like a death sentence for them.

**Reply by datlo** [reply]
You will need to properly prepare to defend the nuke you've set, yes. This is intended.

**Reply by Sadhorizon** [reply]
Thing is - compared to going loud, you are taking more risk, while having less gear. 
It's not like dragging a nuke through whole station unnoticed is easy, so I'd say no announcement for them wouldn't be THAT bad?
And that is if you manage to stealth kill the captain/yoink his bag without him noticing, which is not easy either.


90% of nukie teams find going stealth boring anyways, I dunno if making last 10% that much harder is a good thing

**Comment by CRUNCH-Borg** [comment]
So if the normal nuke is unchanged, why not just steal the normal nuke out of the vault once you have the NAD?

**Comment by Pedritoc25** [comment]
I just found this discussion and really liked it. But iI dont know if someone will work on it
